pkg/objstore/s3/s3.go

```diff
@@ -320,6 +323,7 @@ func configFromEnv() Config {
 	}
 
 	c.Insecure, _ = strconv.ParseBool(os.Getenv("S3_INSECURE"))
+	c.HTTPConfig.InsecureSkipVerify, _ = strconv.ParseBool(os.Getenv("S3_INSECURE_SKIP_VERIFY"))
```

Comment:
Can we document this envvar in `storage.md`?

Reply:
Are the other vars (S3_BUCKET, S3_ENDPOINT, S3_INSECURE, etc.) documented somewhere already? tbh, I'm not totally sure where to document these in storage.md since (afaict) they are only available for e2e tests... is that correct?

Reply:
yup, in `storage.md` (: